src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,27 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, activeTab, setActiveTab }) => {
+  return (
+    <ul>
+      {tabs.map(tab => {
+        return (
+          <li
+            key={tab.id}
+            data-cy="Tab"
+            className={cn({ 'is-active': activeTab === tab.id })}
+          >
+            <a
+              data-cy="TabLink"
+              href={`#${tab.id}`}
+              onClick={() => {
+                setActiveTab(tab.id);
+              }}
```

Comment:
add a condition to not invoke `setActiveTab` if tab.id is equal to activeTab.id. 
And move this onClick logic to an own function

Reply:
ty )